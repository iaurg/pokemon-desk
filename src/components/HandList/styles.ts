import styled from 'styled-components'

export const Wrapper = styled.main``

export const ListContent = styled.div`
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(220px, 1fr));
`
