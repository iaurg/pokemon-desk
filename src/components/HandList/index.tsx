import * as S from './styles'

const HandList = () => (
  <S.Wrapper>
    <h1>HandList </h1>
  </S.Wrapper>
)

export default HandList
