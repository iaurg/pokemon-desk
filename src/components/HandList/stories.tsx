import { Story, Meta } from '@storybook/react'
import HandList from '.'

export default {
  title: 'HandList',
  component: HandList
} as Meta

export const Default: Story = () => <HandList />
