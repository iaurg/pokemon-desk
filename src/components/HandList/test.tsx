import { render, screen } from '@testing-library/react'

import HandList from '.'

describe('<HandList />', () => {
  it('should render the heading', () => {
    const { container } = render(<HandList />)

    expect(
      screen.getByRole('heading', { name: /HandList/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
