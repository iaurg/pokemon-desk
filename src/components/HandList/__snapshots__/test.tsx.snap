// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HandList /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    HandList 
  </h1>
</main>
`
